common/api/summary/presentation-components.exports.csv

```diff
@@ -57,7 +57,7 @@ public;TABLE_DATA_PROVIDER_DEFAULT_PAGE_SIZE = 20
 public;tableWithUnifiedSelection
 public;TableWithUnifiedSelectionProps
 beta;UnifiedSelectionContext
-beta;UnifiedSelectionContextProvider(props: UnifiedSelectionContextProviderProps): React.ReactElement
+beta;UnifiedSelectionContextProvider(props: UnifiedSelectionContextProviderProps): React_2.ReactElement
```

Comment:
Any idea why this shows up as `React_2`?

Reply:
Hmm, I'm not sure why it is auto-creating it that way specially since the cjs created don't have that:
![image](https://user-images.githubusercontent.com/98850418/172685020-e7c92791-3ec9-4b18-923f-cfbeadb3b8b2.png)
![image](https://user-images.githubusercontent.com/98850418/172685156-cd23c081-5604-4242-90e4-8e642f31d55c.png)

Reply:
See also Element_2 and Document_2 in core-backend.

Reply:
Element_2 and Document_2 make more sense to me since those are reserved words within the web already and native types. So technically our "Element" and "Document" are the second type of the same name.

The React one is confusing as I'd only expect one.

Reply:
I have been looking into this and like you mentioned, it doesn't make a whole lot of sense for React to be renamed too since it is not a reserved/exported type in global DOM. There are open issues in the rushstack repo: [1](https://github.com/microsoft/rushstack/issues/2534), [2](https://github.com/microsoft/rushstack/issues/2479), and [3](https://github.com/microsoft/rushstack/issues/2895). Not exactly sure why it is happening but its happening with other imports too such as `Node`, `Text`, `Omit`, `Schema`, `FileReader`. Do you see a pattern here?

This also seems to have been introduced in the newer version of api-extractor. Should I close this PR and perform a force resolution on imodelaccesbackend repo to address the security advisory there instead of trying to perform this upgrade?

Reply:
A not so comprehensive list names that were changed to add `_2` to them:
```text
```

Reply:
I noticed that the `React` imports was using importing type from `@types/react@17.0.45`: 
![image](https://user-images.githubusercontent.com/98850418/175422757-d0b90963-7d00-49ce-ab34-51e5c10bc885.png)

The dependency on the packages however is `@types/react@17.0.37`. I tried changing the dependency version to `17.0.45` and also `17.0.0` (same version as `@types/react-dom`). Neither of this solved the React naming issue we have been seeing. It might not be worth blocking the PR anymore. What do you think @calebmshafer ?

Reply:
There's a way in Rush to force a common version to be used and set for all pkgs in the repo. I just checked the file, https://github.com/iTwin/itwinjs-core/blob/master/common/config/rush/common-versions.json, and it has that version of `@types/react` dependency set so probably the reason for the duplication. I should have realized this a bit earlier!

Reply:
Looks like that wasn't the issue. We're going to ignore the duplicate React type issue for the time being.